drivers/src/ecc384.rs

```diff
@@ -254,7 +255,7 @@ impl Ecc384 {
         };
 
         // Pairwise consistency check.
-        if perform_pct {
+        if cfi_launder(priv_key_usage.ecc_private_key()) {
```

Comment:
I think this branch is too dangerous. Can we just remove the branch and always validate? Why would anyone want to generate a private key that can't be used as a private key?

Reply:
I agree, I can't think of any valid reason why this PCT should ever be skipped.

Reply:
Discussed offline; we'll block creating a key-pair if [ecc_pkey_dest_valid](https://chipsalliance.github.io/caliptra-rtl/main/internal-regs/?p=clp.ecc_reg.ecc_kv_wr_pkey_ctrl#ecc_pkey_dest_valid.desc) is not set